rafiki/admin/admin.py

```diff
@@ -460,13 +517,14 @@ def get_models(self):
                 'datetime_created': model.datetime_created,
                 'user_id': model.user_id,
                 'docker_image': model.docker_image,
-                'dependencies': model.dependencies
+                'dependencies': model.dependencies,
+                'access_right': model.access_right
             }
             for model in models
         ]
 
-    def get_models_of_task(self, task):
-        models = self._db.get_models_of_task(task)
+    def get_models_of_task(self, user_id, task):
+        models = self._db.get_models_of_task(user_id, task)
```

Comment:
TODO: also filter new endpoints `get_models`, `get_model` and `get_model_file` by user, so that private models are not visible to other users

Reply:
Throws an error if user is accessing a private model not belonging to them.